src/components/BookItem.js

```diff
@@ -1,14 +1,28 @@
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../actions';
 
 const BookItem = (props) => {
   const { book } = props;
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = () => dispatch(removeBook(book.id));
```

Comment:
- [ ] This `handleRemoveBook` should be implemented in the BookList component and passed down as a prop to the book component as required. This implementation is creating this function in each book component. Kindly fix this.
![image](https://user-images.githubusercontent.com/57832450/128859033-7a282826-0983-43d9-a538-d566e7786c93.png)